#159 Speed-up dependency computation of 'visibility:reexport' bundles

pdulth opened the pull request.

mickaelistria commented:
I don't understand in which particular case this change is supposed to improve the behavior and how it achieves it. Can you please provide this patch with some more info to understand it. This can be a commit message, more comments, an example, a test... all of them ;)

pdulth commented:
> I don't understand in which particular case this change is supposed to improve the behavior and how it achieves it. Can you please provide this patch with some more info to understand it. This can be a commit message, more comments, an example, a test... all of them ;)

I have updated the commit message and added example in the issue.
Thanks
